h/static/scripts/tests/controllers/character-limit-controller-test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+var CharacterLimitController = require('../../controllers/character-limit-controller');
+var util = require('./util');
+
+describe('CharacterLimitController', function () {
+
+  /**
+   * Make a <textarea> with the character limit controller enhancement
+   * applied and return the various parts of the component.
+   */
+  function component(value, maxlength) {
```

Comment:
Sounds good.

This is one area where CoffeeScript (with its destructuring assignment) really shined.